lib/movie_list.rb

```
@@ -0,0 +1,40 @@
+class MovieList
+  attr_reader :movies
+
+  def initialize(movies = [])
+    @movies = movies
+  end
+
+  def add(movie)
+    if @movies.include?(movie)
```

sure, I can buy that -- if you want `MovieList#add` to not:
1. not add the item if it exists
2. tell the program something bad happened if someone tried to add  a dupe:

in `MovieList#add` I would raise an exception if it is contained when you try to add it.